website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

```diff
@@ -17,7 +17,9 @@ dbt Labs is committed to providing backward compatibility for all versions 1.x,
 
 ## Keep on latest version 
 
-With dbt Cloud, you can get early access to many new features and functionality before they're in the Generally Available (GA) release of dbt Core v1.8 without the need to manage version upgrades. Refer to the [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) setting for more details.
+dbt Cloud is going "versionless." This means you'll automatically get early access to new features and functionality before they're available in final releases of dbt Core. 
+
+Select ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) in your development, staging, and production [environments](/docs/deploy/deploy-environments) to access to everything in dbt Core v1.8 and more.
```

Comment:
adding a small example liner here to try to make it super clear - if you think it's not needed though, I can remove 

```suggestion
Select ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) in your development, staging, and production [environments](/docs/deploy/deploy-environments) to access to everything in dbt Core v1.8 and more.

For example, if the latest version is 1.8, selecting "Keep on latest version" automatically updates your environment to dbt v1.8.
```

Reply:
hey @jtcohen6 i'm going to go ahead and suggest this commit but please let me know if you had other thoughts on it! happy to remove it!